applications/api.py

```diff
@@ -60,7 +58,9 @@ def derive_application_state(
     ]
     if any([status == REVIEW_STATUS_REJECTED for status in submission_review_statuses]):
         return AppStates.REJECTED.value
-    elif any([status is None for status in submission_review_statuses]):
+    elif any(
+        [status == REVIEW_STATUS_PENDING for status in submission_review_statuses]
+    ):
```

Comment:
I think there will only be 1 video submission required at first, but wondering if this should be handled differently at all in the future if/when multiple submissions are required.  For instance, if 3 submissions are required and 2 have been submitted and are pending, is the correct application state `AWAITING_SUBMISSION_REVIEW` or `AWAITING_USER_SUBMISSIONS`?  I'm fine with kicking this can down the road though.

Reply:
I only touched this code because of the migration from a `None` value to the constant `REVIEW_STATUS_PENDING`, so I think this is something for another issue if it's something that needs to be done.